jwala-persistence/src/main/java/com/cerner/jwala/persistence/service/impl/JpaApplicationPersistenceServiceImpl.java

```diff
@@ -178,6 +178,15 @@ public Application updateWarInfo(final String appName, final String warName, fin
     }
 
     @Override
+    public JpaApplication getJpaApplication(String appName){
+        final Query q = em.createNamedQuery(JpaApplication.QUERY_BY_NAME);
+        q.setParameter(JpaApplication.QUERY_PARAM_APP_NAME, appName);
+        final JpaApplication jpaApplication = (JpaApplication) q.getSingleResult();
+        return jpaApplication;
```

Comment:
Inline the return of the jpaApplication variable and remove the newline at the end